src/stylesheets/vendor/slick/slick-active.scss

```diff
@@ -0,0 +1,9 @@
+/* stylelint-disable */
+.slick-active .wheel-select__element {
```

Comment:
@rafayet-monon It should belong to `screen` or `component` 🤔

Reply:
Moved in [7bdd26d](https://github.com/rafayet-monon/surveyor/pull/19/commits/7bdd26dcb7f7a7e4da300145bb73af11b160d4e2)